data/World/Deku Tree MQ.json

```diff
@@ -86,7 +96,10 @@
         "region_name": "Deku Tree Basement Ledge",
         "dungeon": "Deku Tree",
         "locations": {
-            "Deku Tree MQ Deku Scrub": "can_stun_deku"
+            "Deku Tree MQ Deku Scrub": "can_stun_deku",
+            "Deku Tree MQ Basement Recovery Heart 1" : "here(has_fire_source_with_torch) or deku_tree_shortcuts",
+            "Deku Tree MQ Basement Recovery Heart 2" : "here(has_fire_source_with_torch) or deku_tree_shortcuts",
+            "Deku Tree MQ Basement Recovery Heart 3" : "here(has_fire_source_with_torch) or deku_tree_shortcuts"
```

Comment:
These basement recovery hearts also don't need here() b/c both ages can go through the web and collect the heart if they're able to burn it.